cli_tools/diagnostics/main_windows.go

```diff
@@ -284,15 +304,39 @@ func gatherEventLogs(logs chan logFolder, errs chan error) {
 // gatherKubernetesLogs put all the kubernetes log file paths in logFolder channel
 // and errors in error channel.
 func gatherKubernetesLogs(logs chan logFolder, errs chan error) {
-	roots := []string{k8sLogsRoot, crashDump}
-	filePaths, ers := collectFilePaths(roots)
-	for _, err := range ers {
-		errs <- err
+	roots := make([]string, 0)
+	var err error
+	_, err = os.Stat(k8sLogsRoot)
```

Comment:
stat can throw another type of error that you would want to catch as well

Reply:
thanks, fixed